examples/quick_start/scripts/inference/inference.ipynb

```diff
@@ -0,0 +1,530 @@
+{
```

Comment:
<ul><li>We could extract <code>INPUT_FOLDER</code> to a variable, so that users understand that path is something they could change.</li><li>I think that the <code>/workspace</code> is not a standard path inside the container, it is the one you mapped and provided to the <code>preprocessing.py --output_path</code> right? In <code>ranking.md</code> we set <code>OUT_DATASET_PATH=/outputs/dataset</code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/966/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>